custom_components/sonnenbatterie/__init__.py

```diff
@@ -3,7 +3,6 @@
 from .const import *
 import json
 
-# from homeassistant.helpers import config_validation as cv
 # from homeassistant.helpers.entity import Entity
```

Comment:
Wenn du eh dabei bist: In Zeile 29 müsste das `hass.async_add_job(...` ersetzt werden durch ein `await async_create_task(...`, weil wegen deprecated seit HA 2024.4.0

Reply:
Erledigt